Sources/Core/ModernAVPlayer.swift

```diff
@@ -79,9 +112,16 @@ public extension ModernAVPlayer {
         case paused
         case playing
         case stopped
-        case waitingNetwork
+        case waitingForNetwork
         
-        public var description: String { return rawValue.capitalized }
+        public var description: String {
```

Comment:
As you provide a description var for the enum, you can just add `CustomStringConvertible` conformance to the enum.
Even if it's not directly used, it's clearer (and free)
``` swift
enum State: String, CustomStringConvertible {
...
}
```

Reply:
i like it